notebooks/00_core.ipynb

```diff
@@ -30,6 +30,7 @@
     "import os\n",
```

Comment:
<ul><li>i would call this <code>PolynomialRegressor</code>  to match scikit-learn conventions</li><li>if you're feeling adventurous you could subclass <code>BaseEstimator</code>  to get access to scikit-learn's functionality</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>